src/util.cpp

```diff
@@ -1446,11 +1448,20 @@ string FormatFullVersion()
 
 #endif
 
+double Round(double d, int place)
+{
+    const double accuracy = place != 0
+            ? std::pow(10, place)
+            : 1;
+
+    return std::round(d * accuracy) / accuracy;
+}
+
 std::string RoundToString(double d, int place)
 {
     std::ostringstream ss;
-    ss << std::fixed << std::setprecision(place) << d ;
-    return ss.str() ;
+    ss << std::fixed << std::setprecision(place) << d;
```

Comment:
This one is confusing, but after reading 4 manual pages on iostram and iomanipulator I think it does what is expected.
To this point I am not sure whether locale affects the decimal separator.